docs/logical_backup.md

```diff
@@ -84,6 +85,58 @@ spec:
 ```
 By providing the authentication details and the TLS configuration via references to `Secret` keys, this example will store the backups in a local Minio instance.
 
+#### Server-Side Encryption with Customer-Provided Keys (SSE-C)
```

Comment:
This section should go after `Alternatively you can use dynamic credentials from an EKS Service Account using EKS Pod Identity or IRSA:`